dump.c

```diff
@@ -704,6 +704,77 @@ S_opdump_indent(pTHX_ const OP *o, I32 level, UV bar, PerlIO *file,
     va_end(args);
 }
 
+struct OpDumpContext {
+    I32 level;
+    UV bar;
+    PerlIO *file;
+    bool indent_needed;
+};
+
+static void
+S_opdump_print(pTHX_ struct OpDumpContext *ctx, SV *msg)
+{
+    STRLEN msglen;
+    const char *msgpv = SvPV(msg, msglen);
+
+    while(msglen) {
+        if(ctx->indent_needed) {
+            I32 i;
+            PerlIO_printf(ctx->file, "     ");
+
+            for (i = ctx->level-1; i >= 0; i--)
```

Comment:
The declaration of `i` could be moved further down, even into the `for` loop.
```c
for (I32 i = ctx->level-1; ...)
```

Or
```c
I32 i = ctx->level;
while (i-- > 0) {
}
```

Reply:
Ahyes. I just copied that from elsewhere in the file, but yes that would be neater. I've now added a second (well, first) commit to tidy up the place I copied it from to C99 style as well.